### DISCUSSION TITLE
List names don't accept 4-byte Unicode

### DISCUSSION THREAD

**Discussion by 0x6d6c** [community/community#19627] [Other Feature Feedback, Questions, & Ideas]
Hi!
Actually it's **a bug**.

When creating a new list and using a 🅰️ emoji the following error occurs:

![image](https://user-images.githubusercontent.com/4584067/176197155-72169e2c-4f2a-4d36-9a61-e069c1c745fd.png)

**Comment by Maher-Reven** [comment]
@mi6th yeah I've experienced such a situation before.

**Reply by Litlangelok** [reply]
> @mi6th yeah I've experienced such a situation before.

**Comment by viluon** [comment]
Same goes for label descriptions!
![image](https://github.com/community/community/assets/7235381/4a2e5153-4661-4b09-bcd0-cb3e95404b7a)
